appveyor.yml

```diff
@@ -7,9 +7,9 @@ environment:
       PYTHON_ARCH: "32"
       MINICONDA: C:\Miniconda
 
-    - PYTHON: "C:\\Python34"
-      PYTHON_VERSION: "3.4.1"
-      PYTHON_ARCH: "32"
+    - PYTHON: "C:\\Python37-x64"
+      PYTHON_VERSION: "3.7.0"
```

Comment:
We only explicit claim support for Python 3.6, which may change soon; but in the mean time, let's target 3.6.6 here.